sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10367,6 +10368,282 @@
+              "customConditional": "",
+              "logic": [],
+              "attributes": {},
+              "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+              },
+              "type": "htmlelement",
+              "input": false,
+              "tableView": false,
+              "placeholder": "",
+              "prefix": "",
+              "suffix": "",
+              "multiple": false,
+              "defaultValue": null,
+              "protected": false,
+              "unique": false,
+              "persistent": false,
+              "clearOnHide": true,
+              "refreshOn": "",
+              "redrawOn": "",
+              "dataGridLabel": false,
+              "labelPosition": "top",
+              "description": "",
+              "errorLabel": "",
+              "tooltip": "",
+              "hideLabel": false,
+              "tabindex": "",
+              "disabled": false,
+              "autofocus": false,
+              "dbIndex": false,
+              "customDefaultValue": "",
+              "calculateValue": "",
+              "calculateServer": false,
+              "widget": null,
+              "validateOn": "change",
+              "validate": {
+                "required": false,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+              },
+              "allowCalculateOverride": false,
+              "encrypted": false,
+              "showCharCount": false,
+              "showWordCount": false,
+              "allowMultipleMasks": false,
+              "addons": [],
+              "id": "e7vkza"
+            },
+            {
+              "label": "<strong>Are you a student with disabilities and wish to be assessed for disability funding?</strong>",
+              "optionsLabelPosition": "right",
+              "inline": false,
+              "tableView": false,
+              "values": [
+                {
+                  "label": "Yes, I have a disabaility and want this application to be assessed for disability funding.",
+                  "value": "yes",
+                  "shortcut": ""
+                },
+                {
+                  "value": "noIWantToAccessOtherFundingTypesWhileIWaitForAVerificationDecision",
+                  "label": "No, I want to access other funding types while I wait for a verification decision.",
+                  "shortcut": ""
+                },
+                {
+                  "value": "noIDoNotHaveADisability",
+                  "label": "No, I do not have a disability.",
+                  "shortcut": ""
+                }
+              ],
+              "validate": {
+                "required": true,
+                "onlyAvailableItems": false,
```

Comment:
The current builder UI may not have this available but can we change `onlyAvailableItems` to `true`?
We must ensure that it will be validated for any hard-coded list.